### PR TITLE
JMP Author +MSDOS UI

### PR DESCRIPTION
**Pull request by jm-parent**
*Thanks for submitting a PR to the Snppts ✂️ library!*

## Submission description

Everyone loved MS-DOS. 🤫
It was clear, it was beautiful. The MS-DOS UI deserves this tribute <3

msdos_ios_landscape

### Possible platform improvements

It's not impossible that I go further than just the UI part.
Having some old-fashioned interactions and animations can be a good challenge for later

### .NET MAUI Beautiful UI Challenge (August 2022)

If you're submitting this during August 2022, you can enter in the [.NET MAUI Beautiful UI Challenge](https://aka.ms/maui/uichallenge)! After submitting your pull request, grab the URL and fill out the quick form at https://www.surveymonkey.com/r/maui-ui-challenge to let us know where to send your stickers!

i'm on my way ! 
image

### CODE REVIEW

File: src/Snppts/Snippets/MauiMsDosUI.cs [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+using System;
+using Snppts.Extensions;
+
+namespace Snppts.Snippets
+{
+    public class MauiMsDosUI : IAmASnippet
+    {
+        public string Slug => "maui-ms-dos";
+        public string Title => "Maui MS Dos";
+        public string GithubRepoName => "jm-parent/MSDos4Ever";
```
Comment (jfversluis) [inline src/Snppts/Snippets/MauiMsDosUI.cs:13]: Doesn't seem to exist? Is it still set to private?

File: src/Snppts/Snippets/MauiMsDosUI.cs [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+        public IAmAnAuthor AuthorInfo => new JeanMarieParent();
+
+        public IEnumerable<Uri> ImageUris => new List<Uri>
+        {
+            //Mobile
+            new Uri("https://github.com/jm-parent/MSDos4Ever/blob/master/MSDos4Ever/SnpptsImages/msdos_ios_landscape.png?raw=true"),
+            new Uri("https://github.com/jm-parent/MSDos4Ever/blob/master/MSDos4Ever/SnpptsImages/msdos_ios_portrait.png?raw=true"),
+            //Windows
+            new Uri("https://github.com/jm-parent/MSDos4Ever/blob/master/MSDos4Ever/SnpptsImages/msdos_windows.png?raw=true")
```
Comment (jfversluis) [inline src/Snppts/Snippets/MauiMsDosUI.cs:28]: Please make them all portrait, our very advances web application will probably break layout if we stick landscape stuff in it 😅

**Review by jfversluis — CHANGES_REQUESTED** [review]

File: src/Snppts/Snippets/MauiMsDosUI.cs (hunk 1 above)
Comment (jm-parent) [reply src/Snppts/Snippets/MauiMsDosUI.cs:13]: Project set to public ✅

File: src/Snppts/Snippets/MauiMsDosUI.cs (hunk 2 above)
Comment (jm-parent) [reply src/Snppts/Snippets/MauiMsDosUI.cs:28]: I believe in this very advanced web application ! 

Essential UI Kit already tried and it seems to work with landscape images 
image

Comment (jfversluis) [reply src/Snppts/Snippets/MauiMsDosUI.cs:28]: Oh ok thanks! Perfect!

**Review by jfversluis — APPROVED** [review]
Alright, awesome Jean-Marie! Thank you so much for this original contribution 😄 I think you already found the link for the stickers!

**Pull request merged**